tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -2662,6 +2662,15 @@ def _GetMSBuildGlobalProperties(spec, guid, gyp_file_name):
     else:
       properties[0].append(['ApplicationType', 'Windows Store'])
 
+  msvs_windows_sdk_version = None
+  if msvs_windows_sdk_version == None and version.ShortName() == '2017':
+    vs2017_sdk = '10.0.14393.0'
```

Comment:
I don't mind keeping this version. I just wanted to know where this came from.

---
In reply to: [106287872](https://github.com/nodejs/node/pull/11867#discussion_r106287872) [](ancestors = 106287872)